db/migrate/20220921095507_add_coupon_type_and_percentage_rate_to_coupons.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddCouponTypeAndPercentageRateToCoupons < ActiveRecord::Migration[7.0]
+  def change
+    add_column :coupons, :coupon_type, :integer, null: false, default: 0
```

Comment:
Rails/BulkChangeTable: You can use `change_table :coupons, bulk: true` to combine alter queries.